kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/PartitionUtils.java

```diff
@@ -248,6 +251,44 @@ private static Expression rewritePartitionColumnRef(
         return expression;
     }
 
+    /**
+     * Get the target directory for writing data for given partition values.
```

Comment:
Give a simple example? Would really help the reader understand what this method does

Reply:
added